coilsnake/modules/eb/SwirlModule.py

```diff
@@ -46,13 +46,26 @@ def __init__(self):
         self.swirls = []
 
     def read_from_rom(self, rom):
-        self.is_swirl_relocated = test_swirl_relocated(rom)
-
         self.swirl_table.from_block(
             rom, offset=from_snes_address(SWIRL_TABLE_DEFAULT_OFFSET))
 
-        if self.is_swirl_relocated:
-            all_animation_pointers = [entry[0] for entry in RELOCATED_SWIRL_ANIMATION_POINTER_TABLE_POINTER]
+        if test_swirl_relocated(rom):
+            # The modified Rom uses a animation pointer size of 4 instead of 2
+            # TODO: Can we assume the same number of rows? How does the Rom determine this?
```

Comment:
Is there a better way to read in the 4-byte pointer-sized table rather than modifying the eb.yml schema?